notebooks/KWS_Noise_Evaluation.ipynb

```diff
@@ -0,0 +1,905 @@
+    "print(\"Working with device:\", device)\n",
+    "\n",
+    "ai8x.set_device(device=85, simulate=False, round_avg=False)\n",
+    "qat_policy = {'start_epoch': 10, 'weight_bits': 8, 'bias_bits': 8}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = mod.AI85KWS20NetNAS(num_classes=len(classes), num_channels=128, dimensions=(128, 1), bias=True, \n",
+    "                           quantize_activation=False)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "checkpoint = torch.load(trained_checkpoint_path)\n",
+    "\n",
+    "state_dict = checkpoint['state_dict']\n",
+    "new_state_dict = OrderedDict()\n",
+    "for k, v in state_dict.items():\n",
+    "    if k.startswith('module.'):\n",
+    "        k = k[7:]\n",
+    "    new_state_dict[k] = v\n",
+    "checkpoint['state_dict'] = new_state_dict\n",
+    "\n",
+    "ai8x.fuse_bn_layers(model)\n",
+    "ai8x.initiate_qat(model, qat_policy)\n",
+    "model.load_state_dict(checkpoint['state_dict'], strict=False)\n",
+    "\n",
+    "ai8x.update_model(model)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 2: Setting up the Test Set and Parameters"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You need to load the KWS test set, specify a noise list, an SNR level list. \n",
+    "\n",
+    " If you want to specify noise kinds, you should be selecting from the defined classes for MSnoise dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "Processing test...\n",
+      "test set: 11005 elements\n",
+      "Class up (# 31): 425 elements\n",
+      "Class down (# 5): 406 elements\n",
+      "Class left (# 15): 412 elements\n",
+      "Class right (# 23): 396 elements\n",
+      "Class stop (# 27): 411 elements\n",
+      "Class go (# 11): 402 elements\n",
+      "Class yes (# 34): 419 elements\n",
+      "Class no (# 19): 405 elements\n",
+      "Class on (# 21): 396 elements\n",
+      "Class off (# 20): 402 elements\n",
+      "Class one (# 22): 399 elements\n",
+      "Class two (# 30): 424 elements\n",
+      "Class three (# 28): 405 elements\n",
+      "Class four (# 10): 400 elements\n",
+      "Class five (# 7): 445 elements\n",
+      "Class six (# 26): 394 elements\n",
+      "Class seven (# 24): 406 elements\n",
+      "Class eight (# 6): 408 elements\n",
+      "Class nine (# 18): 408 elements\n",
+      "Class zero (# 35): 418 elements\n",
+      "Class UNKNOWN: 2824 elements\n"
+     ]
+    }
+   ],
+   "source": [
+    "sn = SimpleNamespace()\n",
+    "sn.truncate_testset = False\n",
+    "sn.act_mode_8bit = False\n",
+    "\n",
+    "data_path = '/data_ssd'\n",
```

Comment:
should it not be /data?

Reply:
We had been using /data_ssd on our local workstation. I've made the change to /data, thank you.